src/frontend/src/views/GroupMembersPage/index.tsx

```diff
@@ -9,15 +11,39 @@ import {
   StyledTabs,
 } from "./style";
 
-type PrimaryTabType = "members" | "details";
+enum PrimaryTabType {
+  MEMBERS = "members",
+  DETAILS = "details",
+}
 
 export type TabEventHandler = (
   _: React.SyntheticEvent<Record<string, unknown>>,
   tabsValue: never
 ) => void;
 
-const GroupMembersPage = (): JSX.Element => {
-  const [tabValue, setTabValue] = useState<PrimaryTabType>("members");
+interface Props {
+  pathTokens?: string[];
+}
+
+const isValidPrimaryTab = (token: string) => {
+  return token === PrimaryTabType.MEMBERS || token === PrimaryTabType.DETAILS;
+};
+
+const GroupMembersPage = ({ pathTokens }: Props): JSX.Element => {
+  const [primaryQueryParam = "members", secondaryQueryParam = "details"] =
+    pathTokens ?? [];
+  const initialPrimaryTab = (
+    isValidPrimaryTab(primaryQueryParam)
```

Comment:
Nit: Because there's this tertiary that defaults to "members" when invalid, it seems like the default assignment above `[primaryQueryParam = "members", secondaryQueryParam = "details"]` is unnecessary? As it is, there's two places where it defaults to making the primaryQueryParam members -- first in the array when it's missing from pathTokens, and then here when the value is invalid. But since the second default will catch the first case (I'm pretty confident, maybe I'm wrong), it seems a bit confusing to have two defaulting approaches when one would do.

Reply:
... dunno why GitHub added this comment twice.